templates/orders/commission-meta-box-html.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Dokan Template
+ *
+ * Dokan Dashboard Commission Meta-box Content Template
+ *
+ * @var WC_Order $order
+ * @var object $data
+ * @var float $total_commission
+ * @var array $all_commission_types
+ *
+ * @since DOKAN_LITE
+ *
+ * @package dokan
+ */
+
+$total_commission = 0 > $total_commission ? 0 : $total_commission;
+
+$order_total = $data && property_exists( $data, 'order_total' ) ? $data->order_total : 0;
+$net_amount  = $data && property_exists( $data, 'net_amount' ) ? $data->net_amount : 0;
```

Comment:
why are we using `$data && property_exists( $data, 'order_total' ) ? $data->order_total : 0;` instead of using order object?

Reply:
Vai we are getting it from dokan order table.